io/disk/Avago_storage_adapter/avago9361_vd.py

```diff
@@ -99,16 +108,72 @@ def test_maxvd(self):
                     self.vd_details()
                 self.vd_delete()
 
+    def test_init(self):
+
+        """
+        Test case to start Fast and Full init on VD
+        """
+        self.vd_create('WB', 'ra', 'cached', 256)
```

Comment:
I think its better to check the details after each and every time you create a vd, even if its tested before. Never know when it would fail.
Also, consider calling `vd_details()` from `vd_create()`, since you are doing it almost everytime after vd_create().